core/src/translate/ddl.rs

```diff
@@ -72,12 +72,15 @@ pub fn translate_column_def(sql_column_def: &SqlColumnDef) -> Result<ColumnDef>
     })
 }
 
+/// Translate `ColumnOptionDef` to `ColumnOption`.
```

Comment:
Nice comment :+1: , if you change it like this it will take the link.

```suggestion
/// Translate [`SqlColumnOptionDef`] to [`ColumnOption`].
```

<img width="897" alt="image" src="https://user-images.githubusercontent.com/27040963/199691903-e171519b-b77d-4ac2-b43e-8dccccd416d3.png">

Reply:
Ah yes, good point! Thanks a lot 👍